improver_tests/acceptance/SHA256SUMS

```diff
@@ -691,32 +691,32 @@ a723b5cc5a2ba050a33dcc569ba67b93e8043d50bbd868cfcd065d000ae52b39  ./weighted_ble
 850d48dfea1d1e5b696324f0e57a52fce3e3128008c58ccaca82e31598196c67  ./weighted_blending/three_models/ukvx/20190101T0400Z-PT0002H00M-precip_rate.nc
 1eac7df64f49df69aabadc49c7ddaad7d2acd72dc775c99930a963bc6567cb03  ./weighted_blending/weights_from_dict/kgo.nc
-c2b71855f260544ca75aaa5900f3297b58cee1b3b64eec5805662e1927d5fdab  ./wet-bulb-temperature-integral/basic/input.nc
-651c80cd4ab7c720c59c6c0c2c4a2ade85aee81c2ab0c697b577ca87a36d57f2  ./wet-bulb-temperature-integral/basic/with_id_attr/kgo.nc
-644936215dddaa48f1becca9a64baf02193c07fd7b98e8329d1aa2dcda0bff91  ./wet-bulb-temperature-integral/basic/without_id_attr/kgo.nc
-10bd2c82425fafd8d7cc86fb5262d8d6359302f74b79072775fa15ea43a406d7  ./wet-bulb-temperature-integral/realizations/input.nc
-7a223364c7da4ac598de5cde36f12110a01c0bd539e6e34a7ad8cd3e14174e2c  ./wet-bulb-temperature-integral/realizations/kgo.nc
-52de37aa15c072e30817327e40e8d9f0e60924d9164f6458e5633072ae2e04ed  ./wet-bulb-temperature/basic/enukx_pressure.nc
-9e90b526a2355df27d3d29f5548e44d4ff8cae60bea0b9da071e5635a12d7c49  ./wet-bulb-temperature/basic/enukx_relative_humidity.nc
-81d3480537dd30284a9745e152371d2ce529818a5e82fa26cb4d41a5b3969f3b  ./wet-bulb-temperature/basic/enukx_temperature.nc
-7facf1f9031b06bb9384fb2a5c09efb134c95ddee8426d800d8ec92f4acdc11c  ./wet-bulb-temperature/basic/with_id_attr/kgo.nc
-f3b782a8f7bfeb1ce0c5ffbca50b5cc41b02ff1637a1a5e37ec221daf893e36a  ./wet-bulb-temperature/basic/without_id_attr/kgo.nc
-98d9946103314b46d4238e9517fe95f75dd4d1f89218b6eda90a0fbd7ab4bc13  ./wet-bulb-temperature/global/kgo.nc
-09d90a5e08da2546c2022f20d2429acc3d5e40d99f66ea6829b78308b388b6ee  ./wet-bulb-temperature/global/pressure_input.nc
-c100701d272fa226782f9aae296107cc58dfc02cc62cbdf50ebc60007922a787  ./wet-bulb-temperature/global/relative_humidity_input.nc
-d9401dfaf29e1b3d8a5cd84695613d7a7ee0d6ac62c7d157aad1908980f6c122  ./wet-bulb-temperature/global/temperature_input.nc
-bebdae181467d87e770ecd4603fd9b8a162a5c3c86cd5885ebe61c9042adbe3e  ./wet-bulb-temperature/multi_level/enukx_multilevel_pressure.nc
-c44adba807281615d06e282172fac379fc65cc545deb4841222c9adee865d75e  ./wet-bulb-temperature/multi_level/enukx_multilevel_relative_humidity.nc
-c34e4efce804613ffbd8104a36bdefd33432347de66f83494b059ee2b2e8619d  ./wet-bulb-temperature/multi_level/enukx_multilevel_temperature.nc
-0456be85918fd355f02fac1c1bb2568ad7e7907524dd4207b8d1556376215ed0  ./wet-bulb-temperature/multi_level/kgo.nc
+ef62d78071045163858387292ec56c11aa92e1fa6f5580c1e4ac485cd01979fc  ./wet-bulb-temperature-integral/basic/input.nc
+4143e7583d66c6e776e43e619622eab00fdee3293becadfaaa00ddd72147ac6a  ./wet-bulb-temperature-integral/basic/with_id_attr/kgo.nc
+f854f85ef58e2a0dc41c3bcdf940a23daa2782a50d3eaf798335fefa8bea21b8  ./wet-bulb-temperature-integral/basic/without_id_attr/kgo.nc
+ed4e971a00d56d771d79ad5dec90d7be8aa1dafc19811df5734c0d602e5ec57b  ./wet-bulb-temperature-integral/realizations/input.nc
```

Comment:
Is it worth considering thinning these inputs along the vertical coordinate?

Reply:
Not these ones as it will change the result and I don't want to modify the KGO beyond thinning. Thinning vertically would actually change the values, making it harder to demonstrate that the tests are fundamentally unchanged.